File: Programming Language(ver.2)/C#/403.cs
using System;

class TQC403
{
    static void Main()
    {
        char[] arr = new char[100];
        string input = Console.ReadLine();
        arr = input.ToCharArray(0, input.Length);
        for (int i=0; i<input.Length; i++) {
            if (Char.IsLower(arr[i]) == true) {
                arr[i] = Char.ToUpper(arr[i]);
            }
            else {
                arr[i] = Char.ToLower(arr[i]);
            }
        }
        Console.WriteLine(arr);    
    }
}

File: Programming Language(ver.2)/C++/403.cpp
#include <iostream>
#include <cstring>
using namespace std;

int main()
{
    char str[100];
    int i;
    cin >> str;
    for (i = 0; i < strlen(str); i++) {
        if (str[i] >= 'A' && str[i] <= 'Z')
            str[i] = tolower(str[i]);
        else if (str[i] >= 'a' && str[i] <= 'z')
            str[i] = toupper(str[i]);
    }
    cout << str;
    return (0);
}